src/commands/cmd_dispatcher.c

```diff
@@ -72,7 +72,13 @@ static int _read_flags(RedisModuleString **argv, int argc, bool *compact,
 				asprintf(errmsg, "Failed to parse query timeout value");
 				return REDISMODULE_ERR;
 			}
+
+			continue;
 		}
+
+		// reaching here means unkown argument, to avoid sql injections output error
```

Comment:
there might be other reason for reaching this point except for query injection, please rephrase:
```c
// unknown command argument
asprintf(errmsg, "Unknown argument: %s", arg);
return REDISMODULE_ERR;
```